chapter_computer-vision/anchor.md

```diff
@@ -59,7 +107,35 @@ def show_bboxes(axes, bboxes, labels=None, colors=None):
     colors = _make_list(colors, ['b', 'g', 'r', 'm', 'c'])
     for i, bbox in enumerate(bboxes):
         color = colors[i % len(colors)]
-        rect = d2l.bbox_to_rect(bbox.asnumpy(), color)
+        rect = d2l.bbox_to_rect(d2l.numpy(bbox), color)
+        axes.add_patch(rect)
+        if labels and len(labels) > i:
+            text_color = 'k' if color == 'w' else 'w'
+            axes.text(rect.xy[0], rect.xy[1], labels[i],
+                      va='center', ha='center', fontsize=9, color=text_color,
+                      bbox=dict(facecolor=color, lw=0))
+```
+
+```{.python .input}
+#@tab pytorch
+#@save
+def bbox_to_rect(bbox, color):
+    """Convert bounding box to matplotlib format."""
+    return d2l.plt.Rectangle(xy=(bbox[0], bbox[1]), width=bbox[2]-bbox[0], height=bbox[3]-bbox[1], fill=False, edgecolor=color, linewidth=2)
```

Comment:
This method is defined in bounding-box.md
We save it there and use it here.

Similarly for other methods.

Reply:
I added it [here](https://github.com/d2l-ai/d2l-en/pull/1369).